test/run

```diff
@@ -6,6 +6,7 @@
 set -eu
 
 tools/make-bots
+test/common/pixel-tests pull
```

Comment:
Local developers would need that as well, right? We don't ever use test/run locally, and this is easy to forget. In c-machines we have a Makefile rule for it and depend on it by `check`. For cockpit I think it could either be done by autogen.sh (which already installs bots) or test/image-prepare.

Would also be good to add a pointer to test/README.md.

Reply:
Good point!